docs/reference/remap/parse_aws_elb.cue

```diff
@@ -0,0 +1,76 @@
+package metadata
+
+remap: functions: parse_aws_elb: {
+	arguments: [
+		{
+			name:        "value"
+			description: "Access log of the Application Load Balancer."
```

Comment:
I think we may want to call this `parse_aws_alb` (or even `parse_aws_alb_log`) instead then to leave room for parsing classic load balancer (ELB) logs (https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/access-log-collection.html) and network load balancer (NLB) logs (https://docs.aws.amazon.com/elasticloadbalancing/latest/network/load-balancer-access-logs.html) in the future.